tox.ini

```diff
@@ -407,18 +403,16 @@ commands_pre =
               -e {toxinidir}/opentelemetry-semantic-conventions \
               -e {toxinidir}/opentelemetry-sdk \
               -e {toxinidir}/tests/opentelemetry-test-utils \
-  ; opencensus exporter does not work with protobuf 4
```

Comment:
So this is not tested anymore?

Reply:
Right. But I think we can keep the test. Let me commit a solution for that

Reply:
Is this fixed ?

Reply:
@aabmass yes. I added two separated tests—one for otlpexporter and another one for opencensus 🙌🏻